lib/config/fileconf_test.go

```diff
@@ -94,36 +82,39 @@ auth_service:
 
 	// run tests
 	for i, tt := range tests {
-		comment := check.Commentf("Test %v", i)
-
+		comment := fmt.Sprintf("Test %v", i)
```

Comment:
I know this was already the implementation, but can we convert the comments into useful test descriptions instead of using the index:
```go
tests := []struct {
		comment                 string
		inConfigString          string
		outAuthenticationConfig *AuthenticationConfig
	}{
		// ...
		{
			comment: "2 - local auth with u2f",
			`
auth_service:
   authentication:
       type: local
       second_factor: u2f
       u2f:
           app_id: https://graviton:3080
           facets:
           - https://graviton:3080
`,
			&AuthenticationConfig{
				Type:         "local",
				SecondFactor: "u2f",
				U2F: &UniversalSecondFactor{
					AppID: "https://graviton:3080",
					Facets: []string{
						"https://graviton:3080",
					},
				},
			},
		},
		// ...
	// run tests
	for i, tt := range tests {
		comment := tt.comment
```

Reply:
adjusted